pkg/api/v1alpha1/cloudstackmachineconfig_types.go

```diff
@@ -203,7 +203,7 @@ func (c *CloudStackMachineConfig) GetName() string {
 }
 
 func (c *CloudStackMachineConfig) Validate() error {
```

Comment:
Tinkkerbell provider needs to leverage this so the logic won't be missed in provider.